From 873247249112ac26ae02946d3f6a9484a85268a3 Mon Sep 17 00:00:00 2001
From: Jeffrey Cody <jcody@redhat.com>
Date: Tue, 16 Sep 2014 20:11:46 +0200
Subject: [PATCH 08/20] vdi: remove double conversion

Message-id: <843ab27113b00881159ce51034472849d8178804.1410897407.git.jcody@redhat.com>
Patchwork-id: 61212
O-Subject: [PATCH qemu-kvm-rhel RHEL7.1 07/15] vdi: remove double conversion
Bugzilla: 1098086
RH-Acked-by: Fam Zheng <famz@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Max Reitz <mreitz@redhat.com>

From: Paolo Bonzini <pbonzini@redhat.com>

This should be a problem when running on big-endian machines.

Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
Reviewed-by: Benoit Canet <benoit@irqsave.net>
Reviewed-by: Stefan Weil <sw@weilnetz.de>
Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>
(cherry picked from commit 6998b6c11b4e8ca5e3c3e6bdbdf07a4f747d5d37)
Signed-off-by: Jeff Cody <jcody@redhat.com>

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 block/vdi.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/block/vdi.c b/block/vdi.c
index fecfa14..0211023 100644
--- a/block/vdi.c
+++ b/block/vdi.c
@@ -239,7 +239,6 @@ static void vdi_header_to_le(VdiHeader *header)
     cpu_to_le32s(&header->block_extra);
     cpu_to_le32s(&header->blocks_in_image);
     cpu_to_le32s(&header->blocks_allocated);
-    cpu_to_le32s(&header->blocks_allocated);
     uuid_convert(header->uuid_image);
     uuid_convert(header->uuid_last_snap);
     uuid_convert(header->uuid_link);
-- 
1.7.1

